optin/puller/scripts/testnet.env

```diff
@@ -0,0 +1,11 @@
+NETWORK_NAME=testnet
+DATABASE_DIRECTORY=_temp/TESTNET
+SNAPSHOT_HEIGHT=3471104
+
+PREOPTIN_JSON=resources/empty_preoptin.json
+OPTIN_ADDRESS=TCTYYOMGVB6IDGQBKVZOCDDV6MQMM6YZBBK2AT2Z
```

Comment:
`optin_addresses` ;)

Reply:
hm? seems all address parameters are singular

Reply:
but this arg allows multiple addresses, right? (coma separated)